inst/app/app.R

```diff
@@ -0,0 +1 @@
+tfpbrowser::run_app()
```

Comment:
Has this file been added as an entry point in case of deployment on shinyapps.io/rsconnect?